src/NonEmptyArray.ts

```diff
@@ -227,6 +227,21 @@ export function last<A>(nea: NonEmptyArray<A>): A {
   return nea[nea.length - 1]
 }
 
+/**
+ * Get all but the last element of an array, creating a new array.
+ *
+ * @example
```

Comment:
travis is failing because this example doesn't type check:

```ts
// too many parens ----------------------v
assert.deepStrictEqual(init([1, 2]), [1]))
// too many parens ------------------v
assert.deepStrictEqual(init([1]), []))
```

Reply:
Fixed.